common/test/acceptance/pages/lms/courseware.py

```diff
@@ -63,6 +70,22 @@ def xblock_component_html_content(self, index=0):
         """
         return self.q(css=self.xblock_component_selector).attrs('innerHTML')[index].strip()
 
+    def xblock_components_contain_strings(self, expected_component_strings):
```

Comment:
I think this would be easier to read if you called it ```all_xblock_components_contain_strings```. It wasn't clear to me in the usages that they all had to be true, so I had to come back to the doc string to check.